### PR TITLE
Do not indent inside namespaces

### PR DESCRIPTION
**Pull request by kostis**
There is really no need to indent code inside namespaces.

While at it, cleaned up various places in the code (mostly in comments that now fit in single lines).

Before we merge this, we should discuss whether to adopt the C++17 way of writing nested namespaces.

### CODE REVIEW

File: src/synchronization/intranode/mcs_lock.hpp [hunk 1]
```diff
@@ -24,7 +24,7 @@ namespace locallock {
  * @warning The MCS need to be locked and unlocked by the same thread.
  */
 class mcs_lock {
-private:
+ private:
```
Comment (lundgren87) [inline src/synchronization/intranode/mcs_lock.hpp:27]: The private/public(below) indentation is a tab in most other places, which is what I would normally expect.

**Review by lundgren87 — CHANGES_REQUESTED** [review]
One comment left above.

Overall I have no issues with the change of code style but I think that this should be accompanied with a change to `docs/code-style.md`, as this currently states:

> Spacing
> 2. Namespaces, as well as public/private/protected labels should be indented

Namespaces should either simply be removed (if indentation of namespaces is "optional") or added as a third bullet stating that namespaces should not be indented. Optionally, if more namespace and/or code-style related changes are planned this could merit a separate PR.

File: src/synchronization/intranode/mcs_lock.hpp (hunk 1 above)
Comment (kostis) [reply src/synchronization/intranode/mcs_lock.hpp:27]: Let me undo this - as it does not belong in this PR

**Review by lundgren87 — APPROVED** [review]
No more objections from my side. Not sure if anyone else has something to add but I will approve.

**Comment by kostis** [comment]
Any opinions on whether to adopt the C++17 way of writing nested namespaces as part of this PR or not?

**Comment by lundgren87** [comment]
> Any opinions on whether to adopt the C++17 way of writing nested namespaces as part of this PR or not?

I think this is a readability improvement but that it does not necessarily have to be part of this PR. No objection if you want to include it.

**Comment by kostis** [comment]
Let me merge this in order to not unnecessarily delay two other, more important, PRs that I want to submit.

**Pull request merged**